### ISSUE TITLE
Temporal Region Trauma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10105]
**Preferred term label:**
Temporal Region Trauma

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0751184

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Epidural hematoma
(this is my guess as to how Mahsa Amini in Iran died)
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think this would go in Recent past medical history, correct?

**Comment by MickeySegal** [comment]
Yes, anywhere from minutes to days.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000626
- label: Recent temporal region trauma
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: A history of an injury to the temporal region of the skull that occurred recently (several hours, days, or weeks before the current presenting complaint).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10105
- synonyms:
